charts/acapy/README.md

```diff
@@ -109,6 +109,9 @@ API Secret keys expected (unless overridden via secretKeys):
 Seed Secret keys expected:
   seed          (32 char wallet seed when wallet-local-did=true or deterministic DID needed)
 
+Note: In multitenant mode the base wallet should not use a seed. You can explicitly disable the
+seed secret and env var by setting secrets.seed.enabled=false.
```

Comment:
Nit.: might be worth mentioning that the base wallet should NOT use a seed UNLESS this is intentional and the seed is registered beforehand to the ledger the agent will be rooted on.